packages/cli/lib/index.ts

```diff
@@ -112,69 +121,85 @@ program
         tscWatch(debug);
     });
 
-program
-    .command('start')
-    .alias('dr')
-    .alias('docker:run')
-    .description('Run the docker container locally')
-    .action(async function (this: Command) {
-        const { debug } = this.opts();
-        await dockerRun(debug);
-    });
-
-program
-    .command('dev')
-    .alias('develop')
-    .alias('watch')
-    .description('Work locally to add integration code')
-    .option('--no-compile-interfaces', `Watch the ${nangoConfigFile} and recompile the interfaces on change`, true)
-    .action(async function (this: Command) {
-        const { compileInterfaces, autoConfirm, debug } = this.opts();
-        await verifyNecessaryFiles(autoConfirm, debug);
-        if (compileInterfaces) {
-            configWatch(debug);
-        }
-
-        tscWatch(debug);
-        await dockerRun(debug);
-    });
-
 program
     .command('deploy')
-    .alias('d')
-    .alias('deploy:prod')
-    .alias('deploy:cloud')
     .description('Deploy a Nango integration')
-    .option('--staging', 'Deploy to the staging instance')
-    .option('--local', 'Deploy to the local instance')
+    .arguments('environment')
     .option('-v, --version [version]', 'Optional: Set a version of this deployment to tag this integration with. Can be used for rollbacks.')
     .option('-s, --sync [syncName]', 'Optional deploy only this sync name.')
     .option('--no-compile-interfaces', `Don't compile the ${nangoConfigFile}`, true)
-    .action(async function (this: Command) {
+    .action(async function (this: Command, environment: string) {
         const options = this.opts();
         (async (options: DeployOptions) => {
-            const { staging, debug } = options;
-            let env = staging ? 'staging' : 'production';
-            env = options.local ? 'local' : env;
-            await deploy({ ...options, env: env as ENV }, debug);
+            const { debug } = options;
+            await deploy({ ...options, env: 'production' as ENV }, environment, debug);
         })(options as DeployOptions);
     });
 
+// Hidden commands //
+
 program
-    .command('deploy:local')
+    .command('deploy:local', { hidden: true })
```

Comment:
Does it make sense to keep this one? (vs. changing `NANGO_HOSTPORT`)

Reply:
Yeah, I think so. It saves time vs. having to go in and edit the .env file every time for local deployment for us internally.